gapis/replay/builder/builder.go

```diff
@@ -650,6 +655,20 @@ func (b *Builder) RegisterNotificationReader(notificationID uint64, reader Notif
 	return nil
 }
 
+// RegisterReplayStatusReader create and register a NotificationReader, which is used to decode and handle
+// replay status information sent from GAPIR.
+func (b *Builder) RegisterReplayStatusReader(ctx context.Context) error {
+	reader := func(n gapir.Notification) {
+		r := n.GetReplayStatus()
+		label := r.GetLabel()
+		total_instrs := r.GetTotalInstrs()
+		finished_instrs := r.GetFinishedInstrs()
+
+		log.I(ctx, "Replay status: Label: %v; Total instructions: %v; Finished instructions: %v.", label, total_instrs, finished_instrs)
```

Comment:
We probably don't want this in the log by default (Which is what log.I will do). For large traces this will fill the log rather quickly. It would be better to log.D(ctx, ...) which will only show up with debug logging enabled. Any client should be able to get the data from the status package if it cares.